config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy.risk_factors.risk_factors_history = Risk Factors History
+report.pregnancy.risk_factors.risk_factors_history.first_pregnancy = Whether this is the Patient’s first pregnancy
+report.pregnancy.risk_factors.risk_factors_history.previous_miscarriage = Patient has had miscarriages or stillbirths
+
+report.pregnancy.risk_factors.risk_factors_present = Present Risk Factors 
+report.pregnancy.risk_factors.risk_factors_present.primary_condition = Risk factors
+report.pregnancy.risk_factors.risk_factors_present.secondary_condition = Additional Risk factors
+report.pregnancy.risk_factors.risk_factors_present.additional_risk_check = Additional factors that could make this pregnancy high-risk
+report.pregnancy.risk_factors.risk_factors_present.additional_risk = How additional factors that could make this pregnancy high-risk
+report.pregnancy.risk_factors.r_risk_factor_present = Present Risk Factors
+
+report.pregnancy.danger_signs.danger_signs_question_note = Danger signs
+report.pregnancy.danger_signs.vaginal_bleeding = Vaginal bleeding
+report.pregnancy.danger_signs.fits = Fits
+report.pregnancy.danger_signs.severe_abdominal_pain = Severe abdominal pain
+report.pregnancy.danger_signs.severe_headache = Severe headache
+report.pregnancy.danger_signs.very_pale = Very pale
+report.pregnancy.danger_signs.fever = Fever
+report.pregnancy.danger_signs.reduced_or_no_fetal_movements = Reduced or no fetal movements
+report.pregnancy.danger_signs.breaking_water = Breaking of water
+report.pregnancy.danger_signs.easily_tired = Getting tired easily
+report.pregnancy.danger_signs.face_hand_swelling = Swelling of face and hands
+report.pregnancy.danger_signs.breathlessness = Breathlessness
+
+report.pregnancy.safe_pregnancy_practices = Safe Pregnancy Practices
+report.pregnancy.safe_pregnancy_practices.malaria = Malaria
+report.pregnancy.safe_pregnancy_practices.malaria.uses_llin = Uses a long-lasting insecticidal net (LLIN)
+report.pregnancy.safe_pregnancy_practices.iron_folate = Iron Folate
+report.pregnancy.safe_pregnancy_practices.iron_folate.iron_folate_daily = Patient taking iron folate daily
+report.pregnancy.safe_pregnancy_practices.deworming = Deworming
+report.pregnancy.safe_pregnancy_practices.deworming.deworming_med = Received deworming medication
+
+report.pregnancy.safe_pregnancy_practices.hiv_status = HIV Status
+report.pregnancy.safe_pregnancy_practices.hiv_status.hiv_tested =Tested for HIV in the past 3 months
+report.pregnancy.safe_pregnancy_practices.tetanus = Tetanus
+report.pregnancy.safe_pregnancy_practices.tetanus.tt_imm_received = Received any Tetanus Toxoid (TT) immunizations during this pregnancy
+report.pregnancy.safe_pregnancy_practices.safe_pregnancy_practices = Safe pregnancy practices 
+
+report.undo_death_report.patient_display_name = Patient Display Name
```

Is this field even needed?

```suggestion
report.undo_death_report.patient_display_name = Patient Name
```